runbook/KubeDNS-Down.md

```diff
@@ -0,0 +1,30 @@
+# KubeDNS-Down
+
+## Alarm
+```
+KubeDNS-Down 
+```
+
+## Observation
+Run the following command to confirm kube-dns is in the cluster:
+
+`$ kubectl get deployments -n kube-system`
+
+`$ kubectl get pods -n kube-system`
+
+You are looking for the `kube-dns` deployment and pod. 
+
+## Action
+
+If `kube-dns` pod(s) are present but failing, check the logs:
+
+`$ kubectl exec -it kube-dns -n kube-system sh` 
```

Comment:
Shouldn't this be `kubectl logs` instead?
Perhaps `describe pod` as well to check relevant events?